include/riak_cs.hrl

```diff
@@ -518,3 +518,11 @@
 -define(BLOCK_BUCKET_PREFIX, <<"0b:">>).        % Version # = 0
 
 -define(MAX_S3_KEY_LENGTH, 1024).
+
+-ifdef(namespaced_types).
+-type mochiweb_headers() :: gb_trees:tree().
```

Comment:
Not must, but don't you mind updating `mochiweb_headers.erl` to export such types?